hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/SinksTest.java

```diff
@@ -244,7 +246,7 @@ public void mapWithMerging_byName() {
                 srcName,
                 Entry::getKey,
                 Entry::getValue,
-                (oldValue, newValue) -> oldValue + newValue);
+                Integer::sum);
```

Comment:
This was done on purpose due to https://bugs.openjdk.java.net/browse/JDK-8154236